lektor/imagetools.py

```diff
@@ -365,15 +379,15 @@ def process_image(ctx, source_image, dst_filename, width, height=None,
     if height is not None:
         resize_key += 'x' + str(height)
 
-    cmdline = [im, source_image]
+    cmdline = [im, source_image, '-auto-orient']
     if crop:
         cmdline += ['-resize', resize_key + '^',
                     '-gravity', 'Center',
                     '-extent', resize_key]
     else:
         cmdline += ['-resize', resize_key]
 
-    cmdline += ['-auto-orient', '-quality', str(quality), dst_filename]
```

Comment:
Thanks for your feedback, I will fix the points you mentioned. Just for this one: ImageMagick is very picky about the order of arguments:
```
$ convert dsc_tp_8801.jpg -resize 300x400 -auto-orient test.jpg && identify test.jpg
test.jpg JPEG 225x300 225x300+0+0 8-bit sRGB 52.4KB 0.000u 0:00.009
$ convert dsc_tp_8801.jpg -auto-orient -resize 300x400 test.jpg && identify test.jpg
test.jpg JPEG 300x400 300x400+0+0 8-bit sRGB 72.7KB 0.000u 0:00.000
```